packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.yargs.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+
+import fg from 'fast-glob'
+import task, { TaskInnerAPI } from 'tasuku'
+
+import { getPaths } from '@redwoodjs/project-config'
+
+import runTransform from '../../../lib/runTransform'
+
+export const command = 'replace-component-svgs'
+export const description =
+  '(v5.x.x->v6.x.x) Converts imported SVGs used as components to React components'
+
+export const handler = () => {
+  task('Replace Component Svgs', async ({ setOutput }: TaskInnerAPI) => {
+    await runTransform({
+      transformPath: path.join(__dirname, 'replaceComponentSvgs.js'),
+      // Here we know exactly which file we need to transform, but often times you won't.
```

Comment:
```suggestion
```

We use globbing because we **don't** know exactly what files we need to transform, right? So the comment is wrong.